docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -496,30 +496,110 @@ A cicd_task is a single job of ci/cd.
+| `id`                        | varchar  | 255        | This key is generated based on details from the original plugin         | PK      |
+| `rule`                      | varchar  | 255        | The key of the rule that the issue is violating                         |         |
+| `severity`                  | varchar  | 255        | The severity level of the issue                                         |         |
+| `component`                 | varchar  | 255        | The name of the component where the issue was found                     |         |
+| `project_key`               | varchar  | 255        | The key of the project that the issue belongs to                        |         |
+| `line`                      | bigint   |            | The line number where the issue was found                               |         |
+| `status`                    | varchar  | 255        | The status of the issue                                                 |         |
+| `message`                   | longtext |            | The message associated with the issue                                   |         |
+| `debt`                      | bigint   |            | The estimated time required to fix the issue                            |         |
+| `effort`                    | bigint   |            | The effort required to fix the issue                                    |         |
+| `commit_author_email`       | varchar  | 255        | The email address of the author of the commit that introduced the issue |         |
+| `assigne`                   | varchar  | 255        | The person assigned to fix the issue                                    |         |
+| `hash`                      | varchar  | 255        | A hash code for the issue                                               |         |
+| `tags`                      | varchar  | 255        | Any tags associated with the issue                                      |         |
+| `type`                      | varchar  | 255        | The type of the issue                                                   |         |
+| `scope`                     | varchar  | 128        | The scope of the issue                                                  |         |
+| `start_line`                | bigint   | 255        | The starting line of the issue                                          |         |
+| `end_line`                  | bigint   | 255        | The ending line of the issue                                            |         |
+| `start_offset`              | bigint   | 255        | The starting offset of the issue                                        |         |
+| `end_offset`                | bigint   | 255        | The ending offset of the issue                                          |         |
+| `vulnerability_probability` | varchar  | 100        | The probability of the issue being a vulnerability                      |         |

```

Comment:
Where are `vulnerability_probability` and `security_category` from? Issue or hotspot API?